dependencies/README.md

```diff
@@ -0,0 +1,67 @@
+Static Linkage Checker
```

Comment:
Is this the best name for the tool? Should it be something more like CheckStaticLinks? check_static_links?

Reply:
For me, a noun (_static linkage checker_) sounds suitable than verb+objective (_check_static_link_).

Yet, something we can hold affection would be nice. For example _Maven_ is better name than  _Java Dependecy Management Tool_ (hypothetical name).
Now "_Saltlake_" just came up to my mind as the name of the tool. (Static Linkage Checker -> SLC -> Saltlake City -> Saltlake)

Talk to you tomorrow.